src/lit-element.ts

```diff
@@ -194,14 +195,11 @@ export class LitElement extends UpdatingElement {
    */
   protected update(changedProperties: PropertyValues) {
     super.update(changedProperties);
-    const templateResult = this.render() as unknown;
-    if (templateResult instanceof TemplateResult) {
```

Comment:
Does it matter that we're no longer providing a way to tell LitElement not to do any rendering for a particular update? I could see people returning undefined (implicitly) to signal this. It's been a while since I was last working on this PR so I don't remember if / what we decided here.

Reply:
(from offline) Use lit-html's [`noChange`](https://github.com/Polymer/lit-html/blob/21999f5222a26247e442caa73fb6ce0d52410d17/src/lib/part.ts#L49) or [`nothing`](https://github.com/Polymer/lit-html/blob/21999f5222a26247e442caa73fb6ce0d52410d17/src/lib/part.ts#L54) for these cases.